src/SelectPanel/SelectPanel.stories.tsx

```diff
@@ -0,0 +1,88 @@
+import {Meta} from '@storybook/react'
+import React, {useState} from 'react'
+import {theme, ThemeProvider} from '..'
+import {Button} from '../Button'
+import {TriangleDownIcon} from '@primer/octicons-react'
+import {ItemInput} from '../deprecated/ActionList/List'
+import BaseStyles from '../BaseStyles'
+import {SelectPanel} from '../SelectPanel'
+import Box from '../Box'
+
+const meta: Meta = {
+  title: 'Components/SelectPanel',
+  component: SelectPanel,
+  decorators: [
```

Comment:
What do we use the decorators param here for? 🤔

Reply:
I have no idea. I haven't been rewriting the parts that worked as is.

Reply:
No worries!